features/flows/elastic/src/main/java/org/opennms/netmgt/flows/elastic/ElasticFlowRepository.java

```diff
@@ -114,6 +114,21 @@
     private final ClassificationEngine classificationEngine;
 
     private final int bulkRetryCount;
+    
+    private static String eventIndexName = "netflow";
```

Comment:
Per previous review comment, this variable should not be static.

Also, it would probably be better to leave this value as empty string "" by default.